spec/unit/lib/cloud_controller/deployment_updater/updater_spec.rb

```diff
@@ -466,6 +513,26 @@ module VCAP::CloudController
             deploying_web_process.reload.instances
           }.by(1)
         end
+
+        context 'when the max_in_flight is set to 10' do
+          let(:deployment) do
+            DeploymentModel.make(
+              app: web_process.app,
+              deploying_web_process: deploying_web_process,
+              state: 'DEPLOYING',
+              original_web_process_instance_count: original_web_process_instance_count,
+              max_in_flight: 10
+            )
+          end
+
+          it 'scales up the coerced web process by 10' do
+            expect do
+              subject.scale
+            end.to change {
+              deploying_web_process.reload.instances
+            }.by(10)
```

Comment:
similar to my above comment, if `original_web_process_instance_count` is `6` (not sure but from my reading of the test file it seems to be), then I wouldn't expect the actual instances to exceed `6`